src/main/java/com/example/carservice/service/CarServiceImpl.java

```diff
@@ -11,12 +11,26 @@ public class CarServiceImpl implements CarService {
 
     private final CarRepository repository;
 
-    public CarServiceImpl(CarRepository repository) {
+
+    public CarServiceImpl(CarRepository repository, CarRepository carRepository) {
         this.repository = repository;
     }
 
+    public Car findCarByModel() {
+        Car carToUpdate = repository.findCarByModel("IS-250");
```

Comment:
Look at findCarByModel method. What does it return? Maybe we can simplify method  _**public Car findCarByModel()**_?